infrastructure/cdn-in-a-box/README.md

```diff
@@ -182,3 +182,33 @@ Try disabling SELinux or setting it to 'permissive'. SELinux hates letting conta
 ### Traffic Vault container exits with cp /usr/local/share/ca-certificates cp: missing destination
 
 Bring all components down, remove the `traffic_ops/ca` directory, and delete the volumes with `docker volume prune`. This will force the regeneration of the certificates.
+
+## Notes for Apple M1 Pro Configuration
+
+### Code edits in the cdn-in-a-box directory
+
+Re/installed docker from the command line to use user=[your username] flag, and update the dockerfiles in the code for both enroller and traffic_ops in the cdn-in-a-box folder with the correct architecture for the M1. Link to install info for docker: https://docs.docker.com/desktop/install/mac-install/  and use Install from command line. 
+
+- Then update cdn-in-a-box/traffic_ops Dockerfile line 46 to -aarch64/pgdg-redhat-repo-latest.noarch.rpm after EL-${RHEL_VERSION%%.*} 
+- Add on line 18 of the cdn-in-a-box/enroller Dockerfile --platform=linux/amd64 after the 1st FROM.
+ 
+Build and run of it:
+In the trafficcontrol/infrastructure/cdn-in-a-box directory run the following:
+ 
+- make build-builders ~~> this will create all the rpms and copy each rpms into its own folder in the cdn-in-a-box project. This will also create the dist folder under trafficcontrol folder structure even if you deleted yours.
+
+- docker-compose up ~~> this will create docker images, and if rebuild is needed, run docker-compose up --build.
```

Comment:
Because one of the docker files describes a process of copying rpms over, and seems that it's implying it as a manual process. Which does happen if rpms are built using docker-compose build, but when using make, copying and renaming the rpms are done automatically. Hoping this note makes it easier for the next new person! (: